server/statistics/hot_peer_cache.go

```diff
@@ -292,22 +292,30 @@ func (f *hotPeerCache) isRegionHotWithPeer(region *core.RegionInfo, peer *metapb
 	return false
 }
 
-func (f *hotPeerCache) updateHotPeerStat(newItem, oldItem *HotPeerStat, storesStats *StoresStats) *HotPeerStat {
+func (f *hotPeerCache) getHotRegionAntiCount() int {
+	return hotRegionAntiCount + f.getDefaultTimeMedian().GetFilledPeriod()
```

Comment:
will the anti count too large?  in default, it is 4.

Reply:
address